build/networking/README_DOCKER.md

```diff
@@ -313,36 +318,46 @@ ubuntu@vm1:~$
 *NOTE*: If user wants to cleanup, manually remove the qemu VM spawned on the
 machine and delete existing IPDK docker container with command `ipdk rm`
 
-## Section 3: Generating dependent files from P4C and TDI pipeline builder:
+## Section 3: Generating dependent files from P4C and TDI pipeline builder
+
 An open-sourced p4lang P4 compiler is integrated as part of the IPDK container.
-1) The p4c executable is used to generate dependent files.
-   You can execute all these commands on the container.
-    a. export OUTPUT_DIR=/root/examples/simple_l3/
-    b. p4c --arch psa --target dpdk --output $OUTPUT_DIR/pipe --p4runtime-files \
-    $OUTPUT_DIR/p4Info.txt --bf-rt-schema $OUTPUT_DIR/bf-rt.json --context \
-    $OUTPUT_DIR/pipe/context.json $OUTPUT_DIR/simple_l3.p4
-
-2) Steps to generate pipeline binary file:
+
+1. The p4c executable is used to generate dependent files.
+You can execute all these commands on the container.
+
+    ```bash
+      export OUTPUT_DIR=/root/examples/simple_l3/
+
+      p4c --arch psa --target dpdk --output $OUTPUT_DIR/pipe --p4runtime-files
+      $OUTPUT_DIR/p4Info.txt --bf-rt-schema $OUTPUT_DIR/bf-rt.json --context
+      $OUTPUT_DIR/pipe/context.json $OUTPUT_DIR/simple_l3.p4
+    ```
+
+1. Steps to generate pipeline binary file.
 Use tdi_pipeline_builder utility to generate pipeline binary file.
-    a. cd /root/examples/simple_l3/
-    b. tdi_pipeline_builder --p4c_conf_file=simple_l3.conf \
-    --bf_pipeline_config_binary_file=simple_l3.pb.bin
 
-Note: As of today <program>.conf is not generated by the compiler, in that case
-you need to manually update this conf file.
+    ```bash
+      cd /root/examples/simple_l3/
+        
+      tdi_pipeline_builder --p4c_conf_file=simple_l3.conf \
+      --bf_pipeline_config_binary_file=simple_l3.pb.bin
+    ```
+
+Note: As of today < program >.conf is not generated by the compiler, in that case you need to manually update this conf file.
 
 ## Using and compiling included P4 Example pipelines
```

Comment:
You compile the example before you use it, so that verb should go first.
"included" doesn't add meaning and should be omitted.

_Compiling and using P4 Example pipelines_